src/pykeen/models/unimodal/triple_re.py

```diff
@@ -77,25 +77,26 @@ def __init__(
         super().__init__(
             interaction=TripleREInteraction,
             interaction_kwargs=dict(p=p, power_norm=power_norm),
-            entity_representations=EmbeddingSpecification(
+            entity_representations=NodePieceRepresentation(
                 embedding_dim=embedding_dim,
                 initializer=entity_initializer,
                 initializer_kwargs=entity_initializer_kwargs,
                 normalizer=entity_normalizer,
                 normalizer_kwargs=entity_normalizer_kwargs,
             ),
-            relation_representations=[
-                EmbeddingSpecification(
+            relation_representations=[NodePieceRepresentation,NodePieceRepresentation,NodePieceRepresentation],
```

Comment:
I guess those should be shared with the entity repr, so they need to be pre-instantiated

Reply:
https://github.com/LongYu-360/TripleRE-Add-NodePiece/blob/994216dcb1d718318384368dd0135477f852c6a4/TripleRE%2BNodepiece/ogb_wikikg2/model.py#L196-L204

Only the mid part of the relation representations is used for NodePiece entity representations.